generated/nidaqmx/constants.py

```diff
@@ -644,13 +656,13 @@ class SampleInputDataWhen(Enum):
 
 
 class SampleTimingType(Enum):
-    SAMPLE_CLOCK = 10388  #: Acquire or generate samples on the specified edge of the sample clock.
+    SAMP_CLK = 10388  #: Acquire or generate samples on the specified edge of the sample clock.
     BURST_HANDSHAKE = 12548  #: Determine sample timing using burst handshaking between the device and a peripheral device.
     HANDSHAKE = 10389  #: Determine sample timing by using digital handshaking between the device and a peripheral device.
     IMPLICIT = 10451  #: Configure only the duration of the task.
     ON_DEMAND = 10390  #: Acquire or generate a sample on each read or write operation. This timing type is also referred to as static or software-timed.
     CHANGE_DETECTION = 12504  #: Acquire samples when a change occurs in the state of one or more digital input lines. The lines must be contained within a digital input channel.
-    PIPELINED_SAMPLE_CLOCK = 14668  #: Device acquires or generates samples on each sample clock edge, but does not respond to certain triggers until a few sample clock edges later. Pipelining allows higher data transfer rates at the cost of increased trigger response latency.  Refer to the device documentation for information about which triggers pipelining affects. This timing type allows handshaking with some devices using the Pause trigger, the Ready for Transfer event, or the Data Active event. Refer to the device documentation for more information.
+    PIPELINED_SAMP_CLK = 14668  #: Device acquires or generates samples on each sample clock edge, but does not respond to certain triggers until a few sample clock edges later. Pipelining allows higher data transfer rates at the cost of increased trigger response latency. Refer to the device documentation for information about which triggers pipelining affects. This timing type allows handshaking with some devices using the Pause trigger, the Ready for Transfer event, or the Data Active event. Refer to the device documentation for more information.
```

Comment:
old is preferable

Reply:
Updated enum values as in expected constants.py